geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/hash/AbstractHashesIntegrationTest.java

```diff
@@ -199,6 +199,16 @@ public void testHStrLen_failsForNonHashes() {
         .hasMessageContaining("WRONGTYPE");
   }
 
+  @Test
+  public void testHStrlen_givenWrongNumberOfArguments() {
+    assertThatThrownBy(() -> jedis.sendCommand(Protocol.Command.HSTRLEN))
+        .hasMessageContaining("wrong number of arguments");
+    assertThatThrownBy(() -> jedis.sendCommand(Protocol.Command.HSTRLEN, "1"))
+        .hasMessageContaining("wrong number of arguments");
+    assertThatThrownBy(() -> jedis.sendCommand(Protocol.Command.HSTRLEN, "1", "2", "3"))
+        .hasMessageContaining("wrong number of arguments");
+  }
+
```

Comment:
Thanks for adding these! I made this comment in the `HSETNX` PR as well:
I know for most of the other commands (del, getset, exists, etc.) we check the error message more closely, like:
```
.hasMessageContaining("ERR wrong number of arguments for 'get' command");
```
I'm not sure if it's necessary, but maybe we could do the same here to be consistent? I realize there are a few areas where we're still just checking `wrong number of arguments`. We could eventually change those in a different PR.

Reply:
Did we want to change this? Or create a different PR for it? or not change it?

Reply:
Did we want to change this? Or create a different PR for it? Or not change it? Or something else?

Reply:
I second this. It would be pretty awkward to return an error message for the wrong command

Reply:
resolved by f65f4ce